xml/System.Security.Cryptography/RSASignaturePaddingMode.xml

```diff
@@ -35,9 +35,9 @@
 
 `Pkcs1` corresponds to the RSASSA-PKCS1-v1.5 signature scheme of the [PKCS #1: RSA Encryption Standard](http://www.emc.com/emc-plus/rsa-labs/standards-initiatives/pkcs-rsa-cryptography-standard.htm). It is supported for compatibility with existing applications.  
 
-`Pss` corresponds to the Probabilistic Signature Scheme (PSS) of the [PKCS #1: RSA Encryption Standard](http://www.emc.com/emc-plus/rsa-labs/standards-initiatives/pkcs-rsa-cryptography-standard.htm). It is recommended for new applications.
+`Pss` corresponds to the Probabilistic Signature Scheme (RSASSA-PSS) of the [PKCS #1: RSA Encryption Standard](http://www.emc.com/emc-plus/rsa-labs/standards-initiatives/pkcs-rsa-cryptography-standard.htm). It is recommended for new applications.
```

Comment:
@bartonjs I addressed the suggestion you originally gave me in the remark for the PPS enum value by adding your acronym to the general description found in the enum Type. Please let me know if this was the correct change to make.

Reply:
"Probabilistic Signature Scheme" is "PSS".  RSA-signature-using-PSS is "RSASSA-PSS" (RSA Signature Scheme with Appendix  - Probabilistic Signature Scheme).

So where this is used, it was correct as-is, and is incorrect with the change.

If we want to include RSASSA-PSS in the string, it'd be something like

```suggestion
`Pss` corresponds to the RSASSA-PSS signature scheme of the [PKCS #1: RSA Encryption Standard](http://www.emc.com/emc-plus/rsa-labs/standards-initiatives/pkcs-rsa-cryptography-standard.htm). It is recommended for new applications.
```

I'd just leave it as-is.